api/dns/settings.py

```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for dns project.
+
+Generated by 'django-admin startproject' using Django 5.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-pas(cb%&u%sy^elawxq8l#xgr##l(+d7#ho)9@s1ipjcu%)4dt'
```

Comment:
not sure if related to this PR, but maybe get it from an env variable with os.environ.get?

Reply:
Done